components/listingsPage/filters.js

```diff
@@ -271,7 +277,8 @@ const Filters = ({ t }) => {
               <span style={{ fontSize: "14px" }}>
                 single trips
                 <TooltipText
-                  minWidth="300px"
+                  minWidth="230px"
+                  side="bottom"
```

Comment:
This `side` prop is actually responsive so you could do `side={{xs: 'bottom', sm:'left'}}` without needing to change the `minWidth`

Reply:
Ok but @ 300px, it is overflowing no matter the side. On welsh its even more complicated because the words are longer

Reply:
I don't think we need to worry about viewports that small: historically we have considered 320 pixels as a minimum - probably that would be too small now: that's a UX question really. 

But there is a problem still if the `i` icon flows onto a second line, as shown below. That might be considered an edge case though, again one to be dealt with by QA I think. 

![image](https://user-images.githubusercontent.com/6080548/92466988-d1924a00-f1c8-11ea-9c25-8af3d1a13690.png)